src/pmfs/VERSION

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
Where/how is this VERSION (1.0.0) used?

Reply:
My misunderstand for a version control, Just understand it for your point it out. Thanks @brianjmurrell 
 ALL Done.